src/openrct2-ui/windows/MapGen.cpp

```diff
@@ -421,8 +421,8 @@ static void window_mapgen_draw_tab_images(rct_drawpixelinfo* dpi, rct_window* w)
 static int32_t _mapSize = 150;
 static int32_t _baseHeight = 12;
 static int32_t _waterLevel = 6;
-static int32_t _floorTexture = TERRAIN_GRASS;
-static int32_t _wallTexture = TERRAIN_EDGE_ROCK;
+static int32_t _floorTexture = 0;
+static int32_t _wallTexture = 0;
```

Comment:
Sounds a bit weird to drop the define in this case (?)

Reply:
No. There is no guarantee at all that index 0 will be grass and rock. It would be like assuming that ride entry index 0 is always a haunted house.